.travis.yml

```diff
@@ -24,3 +24,9 @@ jobs:
     - name: f30-boot
       before_install: sudo apt-get install -y systemd-container yum qemu-kvm
       script: sudo env "PATH=$PATH" python3 -m test --case f30-boot --build-pipeline samples/build-from-yum.json
+    - name: timezone
+      before_install: sudo apt-get install -y systemd-container yum qemu-kvm curl tar xz-utils
```

Comment:
Do we really need qemu-kvm, curl and xz-utils for these tests?

Reply:
no :)

Reply:
or I should say, once I rebase it on top of your PR, then no